database/seed.js

```diff
@@ -157,31 +157,37 @@ let images = [
   'https://keybox-houses.s3-us-west-1.amazonaws.com/naomi-hebert-MP0bgaS_d1c-unsplash.jpg',
   'https://keybox-houses.s3-us-west-1.amazonaws.com/nuzha-naashidh-D98T9aHmLP0-unsplash.jpg',
   'https://keybox-houses.s3-us-west-1.amazonaws.com/patrick-perkins-3wylDrjxH-E-unsplash.jpg',
-  'https://keybox-houses.s3-us-west-1.amazonaws.com/patrick-perkins-iRiVzALa4pI-unsplash.jpg'
+  'https://keybox-houses.s3-us-west-1.amazonaws.com/patrick-perkins-iRiVzALa4pI-unsplash.jpg',
 ];
 const imageUrl = () => {
-  let index = random(0, images.length, true);
+  const index = random(0, images.length, true);
   return images[index];
-}
+};
 
-const seed = () => {
-  for (let i = 0; i < 20; i++) {
-    Listing.create({
-      superhost: newBoolean(),
-      heart: newBoolean(),
-      reviews: generateReview(),
-      listing: listing(),
-      title: randTitle(),
-      price: randPrice(),
-      image: imageUrl()
-    }, (err, data) => {
-      if (err) {
-        res.status(400).send(err);
-      } else {
-        res.status(200).send(data);
-      }
-    });
+const seed = (/* callback */) => {
+  const results = [];
+  for (let i = 1; i <= 100; i += 1) {
```

Comment:
We only want to generate 100 rooms, and therefore we only need to generate 12 photo objects with the required properties for each of the 100 rooms (eg; superhost, desc, title, etc props on each obj). Think about rewriting this with helper functions, and keeping it DRY.